net/uring/io_uring.c

```diff
@@ -160,3 +160,23 @@ static go_completion_result completion(struct io_uring *ring, int block) {
     io_uring_cqe_seen(ring, cqe);
     return res;
 }
+
+typedef struct {
+  int probingNotSupported;
```

Comment:
This is C, so use snake_case instead of camelCase.

Use positive semantics, so this should be `probing_supported`.